src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle() {
+        this.radius = random.nextInt(VALUE_LIMIT);
+        this.color = colorSupplier.getRandomColor();
+        area();
+    }
```

Comment:
do we need this constructor?
and should constructor without parameters returns figure with random fields?
it isn't obvious behaviour
(for all classes)